README.md

```diff
@@ -1,22 +1,22 @@
-# Bring your Dotfiles to Gitpod
+# Bring your dotfiles to Gitpod
```

Comment:
the title was based on this: [Bring your own (Docker) image to Gitpod](https://www.gitpod.io/blog/docker-in-gitpod)